### ISSUE TITLE
NVDA reads out previous values of table cell after the value is edited to a new value

### ISSUE THREAD

**Issue by Piyush-1107** [nvaccess/nvda#13869]
### Steps to reproduce:
1. Keep the installed NVDA software running.
2. Open this codepen. [Link](https://codepen.io/piyush-1107/pen/RwMWNZP)
3. Fill in the Name and age fields. Fill the age as 12 (for row 2 column 2) and 13 (for row 3 column 2).
4. Now move to row 2 column 2 using mouse, and backspace on "12" to make it "1".
5. Move to row 3 column 2, then again move focus to row 2 column 2, both using mouse.

### Actual behavior:
For the first time, NVDA reads out correct values, that is "12 row 2 column 2 edit 12",
But when after 12 is edited to 1, and the focus is moved to the cell, it reads, "12 row 2 column 2 edit 1". (Notice that 12 is the value of cell before edit)

If the error is not reproduced, try below steps several times:
1. Change the value of one of the Age column cells by adding or deleting a character.
2. Move focus to a different cell, and then back to the edited cell.

### Expected behavior:
NVDA's output should have been "1 row 2 column 2 edit 1".
That is NVDA should narrate the updated value after edit.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA 2022.1

#### Windows version:
Windows 11

#### Name and version of other software in use when reproducing the issue:
Google Chrome Browser Version 103.0.5060.66 (Official Build) (64-bit)

#### Other information about your system:
System: Hp 14s cs1000tu

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?
No

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
No

**Comment by Adriani90** [comment]
We didn't get an update from the initial author since more than one year, I am closing as abandoned. If you are still having this issue, please provide the requested info and we can reopen.
